extensions/wikia/PortableInfobox/services/Parser/Nodes/Node.php

```diff
@@ -71,8 +65,25 @@ public function getType() {
 		return strtolower( $this->xmlNode->getName() );
 	}
 
+	public function isType( $type ) {
+		return strcmp( $this->getType(), strtolower( $type ) ) == 0;
```

Comment:
A nitpick as we don't use diacritics in types at the moment: wouldn't it be safer to use mb_strtolower instead of strtolower?